.github/dependabot.yml

```diff
@@ -0,0 +1,25 @@
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
+  - package-ecosystem: "npm"
+    directory: "/containers/ecr-viewer"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "docker"
+    directory: "/containers/ecr-viewer"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "pip"
```

Comment:
should we also check pip on the fhir converter?

Reply:
Sure! I don't want to add any more though since those others will just get folded into orchestration anyways.